src/main/java/com/ikubinfo/enums/AttributeType.java

```diff
@@ -1,6 +1,17 @@
 package com.ikubinfo.enums;
 
 public enum AttributeType {
-	CABIN, SITE;
 
+	site,
+
+	cabin;
+
+	public static boolean contains(AttributeType type) {
```

Comment:
kjo nuk ben shume sens, sepse ne momentin qe ke nje objekt AttributeType, o eshte null, o eshte cabin, o eshte site. Dmth nuk mund te kesh nje tjeter. Do bente sens nqs do ishte `contains(String type)`, sepse ne momentin qe eshte string mund te jete cdo lloj vlere (psh cabin_site). Nje gje qe duhet te kesh parasysh, eshte qe para se te besh for loop, duhet te kontrollosh qe vlera te mos jete null ose bosh

Reply:
pervec kesaj metode, te sugjeroj te shtosh dhe nje tjeter, `public static AttributeType findByName(String name)` qe ne momentin qe ke vetem name, te gjej AttributeType nga name. (bej nje review te enums ne pergjithesi sepse perdoren goxha)